src/queue/SoundQueue.ts

```diff
@@ -0,0 +1,57 @@
+import config from '../../config/config.json';
+
+import QueueItem from './QueueItem';
+import Util from '../Util';
+
+export default class SoundQueue {
+  private queue: Array<QueueItem>;
+
+  constructor() {
+    this.queue = [];
+  }
+
+  public add(item: QueueItem) {
+    this.queue.push(item);
+  }
+
+  public isStartable() {
+    return this.queue.length === 1;
```

Comment:
Why is a queue of length `2` not startable?

Reply:
See answer to your other comment: The first sound would be interrupted.